static/chats.js

```diff
@@ -8,6 +8,7 @@ let ieraksts = 0
 
 
 let skanjaIeslegta = false
+krasasMaina = (krasa) => document.body.className = krasa
```

Comment:
Šis nebūtu labākais veids. Sanāk, ka mēs pārrakstām visus klašu nosaukumus. Mums pirms tam body jau ir uzlikta klase body (šoreiz gan netiek izmantota), bet potenciāli varētu salauzt dizainu.

Varētu izmantot:
```js
// Pievienojam mūsu klasi
document.body.classList.add("MyClass")
// Noņemam klasi
document.body.classList.remove("MyClass")
```
Šeit gan jāatceras mainīgajā kaut kur iepriekšējā klase (lai zinām, ko noņemt pēc tam nost uz maiņu). Var arī iterēt cauri mūsu kaut kādam krāsu/tēmu masīvam un noņemt to nost.

Reply:
Šo noteikti gribētu pamēģināt izdarīt, bet šovakar tam diemžēl nebūs laika.